index.html

```diff
@@ -4,44 +4,111 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Moore Student Success Website</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/4e37bbbb15.js" crossorigin="anonymous"></script>
 </head>
+
 <body>
+    <article>
 <header>
     <nav>
-        <img src="images/mss-logo.jpg" alt="moore student success logo">
-        <ul>
-            <li><a href="index.html">Home</a></li>
-            <li><a href="about.html">About</a></li>
-            <li><a href="services.html">Services</a></li>
-            <li><a href="contact.html">Contact</a></li>
+        <div>
+<ul class="menu">
+    <li class="logo"><a href="index.html"> <img src="images/mss-logo.jpg" alt="moore student success logo"></a></li>
+    <li class="item"><a href="index.html">Home</a></li>
+    <li class="item"><a href="about.html">About</a></li>
+   <li class="item has-submenu">
+    <a tabindex="0">Services <i class="fa-solid fa-angle-down"></i></a>
+        <ul class="submenu">
+            <li class="subitem"><a href="editing.html">Editing</a></li>
+            <li class="subitem"><a href="proofreading.html">Proofreading</a></li>
+            <li class="subitem"><a href="formatting.html">Manuscript Formatting</a></li>
+            <li class="subitem"><a href="tutoring.html">Tutoring</a></li>
+            <li class="subitem"><a href="curriculum.html">Curriculum Development</a></li>
+            <li class="subitem"><a href="research.html">Research Assistance</a></li>
+            
         </ul>
+    </li>
+    <li class="item"><a href="contact.html">Contact</a></li> 
+    <li class="toggle"><i class="fa-solid fa-bars"></i></li>
+</ul>
+</div>
     </nav>
 </header>    
 <main>
-    <div> 
-        <h1>Testimonials</h1>
-        <p>Testimonial from a client</p>
+   <div class="slideshow-container">
```

Comment:
missing an h1 on this page. you can hide it visually if you don't want it displayed, but it needs to exist. see https://www.a11yproject.com/posts/how-to-hide-content/

Reply:
Fixed h1 on index page.